tools/Version.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "Collector": "2.0.14",
+    "Collector": "2.0.15",
     "Analyzer": "2.0.11",
     "Generator": "2.0.5"
```

Comment:
```suggestion
    "Collector": "2.1.14",
    "Analyzer": "2.1.11",
    "Generator": "2.1.5"
```

Reply:
These should at least be a minor version and not a just a patch

Reply:
Need change `$Script:Version` in the ps1 scripts too if this suggestion applied.